tests/x11/smt.pm

```diff
@@ -57,7 +58,7 @@ sub run() {
     }
     assert_script_run 'smt-sync', 600;
     assert_script_run 'smt-repos';
-    type_string "killall xterm\n";
+    send_key "alt-f4";
```

Comment:
see cc19fd6 regarding this change and [poo#10646](https://progress.opensuse.org/issues/10646). If there is a good reason for using 'alt-f4' instead then the same should be used all over the place where 'killall xterm\n' is also used.